src/clients/compositor/mod.rs

```diff
@@ -54,6 +54,26 @@ impl Compositor {
         }
     }
 
+    pub fn create_keyboard_layout(
```

Comment:
Should be renamed to match existing method naming

```suggestion
    pub fn create_keyboard_layout_client(
```